microservices-orders/docs/index.rst

```diff
@@ -363,9 +373,10 @@ Exercise 3: Enriching streams with joins
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Streams can be enriched with data from other streams or tables through joins.
-Many stream processing applications in practice are coded as streaming joins.
-For example, applications backing an online shop might need to access multiple updating database tables (e.g., sales prices, inventory, customer information) in order to enrich a new data record (e.g., customer transaction) with context information.
-In these scenarios, you may need to perform table lookups at very large scale and with a low processing latency.
+A join enriches data by performing lookups in a streaming context where data is updated continuously and concurrently.
+For example, applications backing an online retail store might enrich new data records with information from multiple databases.
+In that scenario, it may be that a stream of customer transactions is enriched with sales price, inventory, customer information, etc.
```

Comment:
```suggestion
In this scenario, it may be that a stream of customer transactions is enriched with sales price, inventory, customer information, etc.
```